torch/distributed/checkpoint/default_planner.py

```diff
@@ -92,12 +151,42 @@ def transform_object(self, write_item: WriteItem, object: Any):
 
 
 class DefaultLoadPlanner(LoadPlanner):
+    """
+    LoadPlanner that adds multiple features on top of DefaultLoadPlanner.
```

Comment:
is this comment accurate? shouldn't it be sth reverse?

```suggestion
    DefaultLoadPlanner that adds multiple features on top of LoadPlanner.
```

Reply:
Thanks for catching.